nilearn/decomposition/tests/test_canica.py

```diff
@@ -3,17 +3,34 @@
 import numpy as np
 from numpy.testing import assert_array_almost_equal
 from nose.tools import assert_true, assert_raises
-
 import nibabel
-
 from nilearn.decomposition.canica import CanICA
 
 
-def test_canica_square_img():
-    shape = (20, 20, 1)
-    affine = np.eye(4)
-    rng = np.random.RandomState(0)
+try:
+    from nose.tools import assert_less_than
+except ImportError:
+    def assert_less_than(a, b):
+        if a > b:
+            raise AssertionError("%f is not less than %f" % (a, b))
+
```

Comment:
Just use assert_true I would say. If we care about providing an assert_less, it should be done somewhere in _utils rather than in one test file.

Two comments:
* assert_less_than does not exist it seems (it is called assert_less AFAICT)
* what you implemented is actually assert_less_equal